packages/contracts/tasks/deploy.ts

```diff
@@ -99,6 +99,12 @@ task('deploy')
     undefined,
     types.string
   )
+  .addOptionalParam(
```

Comment:
Why is it optional? What is the default?

Reply:
Note that we have now have gotten rid of the `+1` in `10**8+1` - see https://github.com/ethereum-optimism/optimism/issues/823#issuecomment-849189622